config/checkstyle/checkstyle-suppressions.xml

```diff
@@ -48,6 +48,7 @@
     <suppress files="client[\\/]" checks="ExecutableStatementCount"/>
     <suppress files="api[\\/]" checks="ExecutableStatementCount"/>
     <suppress files="BlueprintTestBase" checks="LineLength"/>
+    <suppress checks="ParameterNumber" files="DatalakeDrClient.java"/>
```

Comment:
@RokolyA94 Because the two functions (triggerBackup and triggerRestore) I modified in that java file have 8 parameters, which exceed the the max limitation. I don't think there's a way to just suppress a function in a file? If there's one, plz let me know! Another way we can do is we can pass in `DatalakeDrSkipOptions` in the two functions rather than listing every skip out, but Im not seeing any `package com.sequenceiq.datalake.flow.dr` being imported in `datalake-dr-connector` so I was thinking that may not be the best way.

Reply:
@RokolyA94 After consideration I think it would look better to pass in `DatalakeDrSkipOptions ` in those functions so we would not have that many parameters, however I was not able to make it work (it seems I need to add a gradlew dependency). I may ask you for help tomorrow... Thank you!!

Reply:
it is possible to suppress a warning with an annotation `@SuppressWarnings("ParameterNumber")` but in this case it is better to use `DatalakeDrSkipOptions` instead of passing one-by-one.

Reply:
Hi @RokolyA94 @votez I removed the suppression and added in `DatalakeDrSkipOptions ` as an argument to decrease the parameter number. After discussing with Sergiy, I have to move that class to be under `datalakake-dr-connector` though otherwise it can not be imported. Please let me know if the update looks good. Thank you so much!